### PR TITLE
feat(icons): Add `cloud-alert` icon

### PR DESCRIPTION
**Pull request by lscheibel**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description

Adds new cloud-alert icon

### Icon use case 

My use case is for a "sync with cloud" feature, where I show the icon similar to how Google docs does it, when there is an issue with syncing to the cloud.

Another use case might be an extreme weather warning. 

### Alternative icon designs 

I don't have alternative designs.

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: cloud, circle-alert
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by lscheibel** [comment]
Related:
https://github.com/lucide-icons/lucide/issues/2055

File: icons/cloud-alert.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 12v4" />
+  <path d="M12 20h.01" />
+  <path d="M17 18h.5a1 1 0 0 0 0-9h-1.79A7 7 0 1 0 7 17.708" />
```
[the remark this answers is not in the document]
Comment (lscheibel) [reply icons/cloud-alert.svg:14]: The 3px gap was intentional as it pulls the exclamation mark together as one element and separates it from the cloud. Note how now the distance between the stem and dot of the exclamation mark is the same as the distance to the cloud.

Btw extremely thankful for lucide-studio! Absolutely insane work! I think you should promote it more in the docs :)

[the remark this answers is not in the document]
Comment (jguddas) [reply icons/cloud-alert.svg:14]: Fair enough.

File: icons/cloud-alert.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "lscheibel"
```
Comment (jguddas) [inline icons/cloud-alert.json:4]: You should inherit the contributors from the icons you based this on.

**Review by jguddas — CHANGES_REQUESTED** [review]

File: icons/cloud-alert.json (hunk 2 above)
Comment (lscheibel) [reply icons/cloud-alert.json:8]: Done! Also added them to the cloud-check icon

**Review by jguddas — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Comment by ericfennis** [comment]
Nice icon!

**Pull request merged**